packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1634,6 +1621,31 @@ class TextSelectionGestureDetectorBuilder {
         && renderEditable.selection!.end >= textPosition.offset;
   }
 
+  /// Returns true if position was on selection.
+  bool _lastTapWasOnSelection(Offset position, TextSelection? targetSelection) {
```

Comment:
Nit: Maybe change the name of the method since `position` doesn't necessarily have to be the last tap?